src/Framework/Framework/Hosting/DotvvmPresenter.cs

```diff
@@ -447,6 +454,11 @@ public async Task ProcessStaticCommandRequest(IDotvvmRequestContext context)
                 
                 return commandResultOrNotYetComputedAwaitable;
             }
+            catch (TargetInvocationException e) when (e.InnerException is DotvvmInvalidStaticCommandModelStateException ex)
```

Comment:
Move this to the `(Exception ex)` branch after unwrapping the `TargetInvocationException`.